### PR TITLE
TDI-43723 - Updating Commons Beanutils + Xerces

### PR DESCRIPTION
**Pull request by coheigea**
https://jira.talendforge.org/browse/TDI-43723

I also removed a duplicate dependency declaration to fix a maven build warning.

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/247/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/247/)

File: services/components-api-service-rest/pom.xml [hunk 1]
```diff
@@ -209,12 +209,6 @@
             <version>${components.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.talend.components</groupId>
-            <artifactId>components-jdbc-runtime</artifactId>
-            <version>${components.version}</version>
-            <scope>test</scope>
-        </dependency>
```
Comment (chmyga) [inline services/components-api-service-rest/pom.xml]: Why do you remove jdbc?

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/286/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/286/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3ATDI-43723)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/297/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/297/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/298/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/298/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3ATDI-43723)

**Pull request merged**